libyul/backends/evm/EVMObjectCompiler.cpp

```diff
@@ -64,7 +64,7 @@ void EVMObjectCompiler::run(Object& _object, bool _optimize)
 	yulAssert(_object.code, "No code.");
 	// We do not catch and re-throw the stack too deep exception here because it is a YulException,
 	// which should be native to this part of the code.
-	CodeTransform transform{m_assembly, *_object.analysisInfo, *_object.code, m_dialect, context, _optimize};
+	CodeTransform transform{m_assembly, *_object.analysisInfo, *_object.code, m_dialect, context, _optimize, {}, true};
```

Comment:
Ah, so that's where the meat of this PR is hiding :) Missed that initially because the `true` at the end is not very self-explanatory.

```suggestion
	CodeTransform transform{
		m_assembly,
		*_object.analysisInfo,
		*_object.code,
		m_dialect,
		context,
		_optimize,
		ExternalIdentifierAccess{},
		true /* _useNamedLabelsForFunctions */
	};
```

So I guess this just enables the feature implemented earlier. Seems pretty straightforward. What would be the reasons against doing it?